bom/runtime/pom.xml

```diff
@@ -1791,6 +1791,20 @@
                 <groupId>org.testcontainers</groupId>
                 <artifactId>testcontainers</artifactId>
                 <version>${test-containers.version}</version>
+                <exclusions>
```

Comment:
These are no longer in the dependency list:
https://mvnrepository.com/artifact/org.testcontainers/testcontainers/1.14.1

Reply:
Thanks for pointing that out, I will update the PR